plasmapy/atomic/particles.py

```diff
@@ -119,10 +119,13 @@ def _create_Particles_dict() -> typing.Dict[str, dict]:
                 Particles[thing]['mass'] = 3.167_47e-27 * u.kg
                 Particles[thing]['half-life'] = 2.906e-13 * u.s
 
-    for thing in ['p', 'p-']:
+    for thing in _neutrinos + _antineutrinos:
+        Particles[thing]['mass'] = None
```

Comment:
I'm not sure why we're doing this.

Reply:
I'm still not really sure why we're doing this. Neutrinos aren't massless, right? I remember hearing about that on A Capella Science. Is `mass` the mass number?